wiki/Manual.md

```diff
@@ -5953,109 +5953,26 @@ class BaseError (Exception):
 class BaseError extends \Exception {}
 ```
 
-Below is an outline of exception inheritance hierarchy:
+Here is an outline of exception inheritance hierarchy: https://github.com/ccxt/ccxt/blob/master/ts/src/base/errorHierarchy.ts
 
-```text
```

Comment:
we'd better get rid of such ASCII drawing, which takes much space and it's even being manually maintained here. make users to look into the self-explanatory JSON tree, anyone would be able to read there.

Reply:
but if you look at it closely, it's harder to read (with some plus and half-cut lines) than JSON...

Reply:
well.. as you say,ok.